Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG package=arcaflow_plugin_utilities
 # STAGE 1 -- Build module dependencies and run tests
 # The 'poetry' and 'coverage' modules are installed and verson-controlled in the
 # quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
-FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.3.1 as build
+FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.4.0 as build
```

Comment:
The Renovate bot updated the tag for `arcaflow-plugin-baseimage-python-osbase` (at line 29) but not this tag; we should update this tag (or wait for the bot to do it) before merging this PR.

Reply:
The Renovate bot has run again and addressed this.